hibernate/src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,102 @@
+package org.example;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.cfg.Configuration;
+
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Configuration configuration = Connection.getConfiguration();
+//       create(configuration);
+//        update(configuration);
+//        delete(configuration);
+//        findForId(configuration);
+        createNewOrder(configuration);
+
+    }
+
+    public static void create(Configuration configuration) {
```

Comment:
So, here after changes I wrote, remove from methods parameters "Configuration configuration" and use try-with-resources to open Session using getSession() methods

<img width="349" alt="image" src="https://github.com/MarynaSkrypnyk/a-level/assets/72043323/5a194f91-2adf-4ab9-ad92-e70ec3061ca9">